indigo_app/views/tasks.py

```diff
@@ -52,13 +55,15 @@ def get(self, request, *args, **kwargs):
 
     def get_queryset(self):
         tasks = Task.objects.filter(country=self.country, locality=self.locality).order_by('-updated_at')
-        return self.form.filter_queryset(tasks, frbr_uri=self.request.GET.get('frbr_uri'))
+        return self.form.filter_queryset(tasks, frbr_uri=self.kwargs.get('frbr_uri'))
```

Comment:
As mentioned above with the FRBR URI, I think it's simpler to make the work just another filter. This would allow selecting multiple works, too.

Reply:
My understanding was that we'd use workflows to group tasks that belonged to certain works together, but that we'd sometimes want to see all the tasks on a work to see what had already been resolved / still needed to be dealt with, especially before publishing.
I also think that if we had to have all the works for a place as a multi select dropdown in the task listing view that could get pretty hairy pretty quickly.

Let's chat about this before I make any other changes to make sure we're on the same page?